_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Recommendations Report
+date: 2019-08-02
+authors:
+  - glen-thurston
+tags:
+  - blog post
+  - open source
+  - rates rebate
+  - te whakamāmā i ngā reiti
+excerpt: "Following a successful Alpha phase in May and September 2018, the Rates Rebate beta trial was run in June and July 2019 by a team with members from Service Delivery and Operations team within DIA, Government Information Services, and the Service Innovation Lab. This was in collaboration with Tauranga City Council, Hutt City Council, and Kāpiti Coast District Council"
+---
+## Executive Summary
+
+A Rates Rebate is a subsidy (currently up to $640) for low-income homeowners on the cost of their residential rates. The subsidy is delivered by Local Councils and is administered by Central Government (currently the Department of Internal Affairs (DIA)). The entitlement value is calculated by the cost of the rates, applicants income, and number of dependants. The maximum subsidy amount is reviewed yearly.
+
+Following a successful (and [award winning](https://nzosa.org.nz/finalists2018/)) [Alpha](https://www.digital.govt.nz/blog/making-it-easier-the-rates-rebates-story-so-far/) phase in May and September 2018, the Rates Rebate beta trial was run in June and July 2019 by a team with members from Service Delivery and Operations team within DIA, Government Information Services, and the Service Innovation Lab.
+
+The Service Innovation Lab invited three councils (Tauranga City Council, Hutt City Council, and Kāpiti Coast District Council) to participate from a larger group that expressed interest. The three councils each run a different IT system for processing applications, providing a good test of any proposed process changes, and a good indication of the challenges of rolling out the solution more widely.
+
+Evidence gathered during the beta trial supported the observation during the Alpha phase that an optimised online application form takes just 4 minutes to complete on average, compared to 25 minutes for the paper form. Feedback from councils supported evidence gathered via observation that the online application was considered much easier for applicants to complete.
```

Comment:
"feedback ... was considered" is passive. Can we make it active by saying who considered the feedback?